pkg/action/executor/workload_controller_resizer.go

```diff
@@ -282,13 +170,14 @@ func resizeWorkloadController(clusterScraper *cluster.ClusterScraper, ormClient
 	return nil
 }
 
-func getContainerIndex(pod *k8sapi.Pod, containerName string) int {
+func getContainerIndex(podSpec *k8sapi.PodSpec, containerName string) int {
 	// We assume that the pod spec is valid.
-	for i, cont := range pod.Spec.Containers {
+	for i, cont := range podSpec.Containers {
 		if cont.Name == containerName {
 			return i
 		}
 	}
 
+	glog.V(4).Infof("Match for container %s not found in pod template spec : %v", containerName, podSpec)
```

Comment:
Do we want to show such message as Error instead of hiding it in V4 info? Looks normally it's not expected to miss the target container in pod template spec.